internal/lsp/completions/providers/regov1.go

```diff
@@ -12,29 +12,21 @@ type RegoV1 struct{}
 func (*RegoV1) Run(c *cache.Cache, params types.CompletionParams, _ *Options) ([]types.CompletionItem, error) {
 	fileURI := params.TextDocument.URI
 
-	fileContents, ok := c.GetFileContents(fileURI)
-	if !ok {
-		// if the file contents is missing then we can't provide completions
-		return nil, nil
+	lines, currentLine := completionLineHelper(c, fileURI, params.Position.Line)
```

Comment:
This is refactoring  completionLineHelper shares some code needed in most completion providers.